group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -256,6 +259,40 @@ public HeartbeatResponseData genericGroupHeartbeat(
         );
     }
 
+    /**
+     * Fetch offsets for a given Group.
```

Comment:
nit: should the comment mention this method fetches offsets for a given set of topics? I realize it is in the parameters, but could be useful to more easily distinguish this and fetchAllOffsets